<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Planning 1.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/Resources/appIcon.png</normaloff>:/Resources/appIcon.png</iconset>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow {
	background-color: rgb(230, 240, 249);
}

QDialog{
background-color: rgb(230, 240, 249);
 }

QToolButton{
background-color: rgb(230, 240, 249);
border-style: none;
font: 8pt &quot;Open Sans&quot;;
}

QPushButton {
background-color:  qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                	   stop: 0 			   #FF007F96, stop: 0.614 white,
                                	   stop: 0.614   #7aacd7,  stop: 0.853 white,
										   stop: 0.853 	#FF007F96, stop: 1.0 #FF007F96);
border-style: none;
border-radius: 5px;
color: white;
min-height: 2em;
min-width: 6em;
font: 10pt &quot;Open Sans&quot;;
}

QPushButton::hover {
background-color:  qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                	   stop: 0 			   #c60c30,  stop: 0.614 white,
                                	   stop: 0.614   #e5506c, stop: 0.853 white,				/*e0415f*/
										   stop: 0.853 	#c60c30,  stop: 1.0 #c60c30);
border-style: none;
border-radius: 5px;
color: white;
min-height: 2em;
min-width: 6em;
font: 10pt &quot;Open Sans&quot;;
}

QPushButton::disabled {
background-color: rgb(155, 155, 155);
border-style: none;
border-radius: 5px;
color: white;
min-height: 2em;
min-width: 6em;
font: 10pt &quot;Open Sans&quot;;
}

QMessageBox QLabel{
font: 14px &quot;Open Sans&quot;;
color: rgb(88,88,90);
}

QDialog QLabel{
color: rgb(88,88,90);
}

QTextEdit{
font: 10pt &quot;Open Sans&quot;;
border-style: none;
color: rgb(29,40,86);
}

QTabWidget::pane { /* The tab widget frame */
    border-top: 1px solid #C2C7CB;
}

QTabWidget::tab-bar {
    left: 5px; /* move to the right by 5px */
}

/* Style the tab using the tab sub-control. Note that
    it reads QTabBar _not_ QTabWidget */
QTabBar::tab {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
    border: 2px solid #C4C4C3;
    border-bottom-color: #C2C7CB; /* same as the pane color */
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
	color : black;
    min-width: 8ex;
    padding: 2px;
}

QTabBar::tab:selected
{
	    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 			#7aacd7, stop: 0.614 white,
                                stop: 0.614  #FF007F96,  stop: 0.853 white,
									stop: 0.853 	#7aacd7, stop: 1.0 #7aacd7);
		color : white;
}

QTabBar::tab:hover 
{
	background-color:  qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                	   stop: 0 			   #c60c30,  stop: 0.614 white,
                                	   stop: 0.614   #e5506c, stop: 0.853 white,				
										   stop: 0.853 	#c60c30,  stop: 1.0 #c60c30);
		color : white
}

QTabBar::tab:selected {
    border-color: #7aacd7;
    border-bottom-color: #7aacd7; /* same as pane color */
}

QTabBar::tab:!selected {
    margin-top: 2px; /* make non-selected tabs look smaller */
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QToolButton" name="tbImport">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Import</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{background-color : transparent;}</string>
    </property>
    <property name="text">
     <string>Import</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/Resources/import.ico</normaloff>:/Resources/import.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="tbSettings">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{background-color : transparent;}</string>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/Resources/settings.png</normaloff>:/Resources/settings.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="tbCharts">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{background-color : transparent;}</string>
    </property>
    <property name="text">
     <string>Charts</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/Resources/charts.png</normaloff>:/Resources/charts.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="tbPlanning">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{background-color : transparent;}</string>
    </property>
    <property name="text">
     <string>Planning</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/Resources/planning.png</normaloff>:/Resources/planning.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="tbHelp">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{background-color : transparent;}</string>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/Resources/help.png</normaloff>:/Resources/help.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>28</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="styleSheet">
    <string notr="true">QToolBar{background-color : transparent};</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="workSpaceDockWidget">
   <property name="styleSheet">
    <string notr="true">QDockWidget{border-width: 1px; 
					 border-style:  solid;
					 border-color:  white};</string>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>WorkSpace</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <property name="styleSheet">
     <string notr="true">QWidget{border-width: 1px; 
					 border-style:  solid;
					 border-color:  white};</string>
    </property>
   </widget>
  </widget>
  <action name="actionInport">
   <property name="text">
    <string>Inport</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
